### ISSUE TITLE
Consider making transfer widgets hideable

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/namecoin-core#529]
Currently, the Configure Name dialog always has the Transfer widgets visible, with the expectation that the user will leave the Transfer address widget blank if they do not intend to make a transfer.  This may be a source of unnecessary cognitive overhead.  We may want to consider only showing an unticked "Transfer name" tickbox by default, and only show the address widget if the user ticks the box.

**Comment by rehrar** [comment]
Yes, absolutely. Excess options that are used less often than others should, as a general rule, be hidden from the main dialog so as not to add extra cognitive load.

**Comment by JeremyRand** [comment]
@domob1812 Any objection?  If this is OK with you, I can implement it.

**Comment by JeremyRand** [comment]
I mean, if we wanted to support transferring names in bulk (which is not a bad idea IMO), I'm not convinced that rejecting this proposal is the right way to do it.  Seems that a distinct dialog that omits the Data field, allows pasting a list of addresses, and transfers multiple names at once, would be more efficient for everyone.  The Configure Name dialog is very clearly optimized for single-name operations.

**Comment by domob1812** [comment]
ConceptACK from me.
